### DISCUSSION TITLE
PR merge into custom base branch

### DISCUSSION THREAD

**Discussion by the-voidl** [renovatebot/renovate#21499] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe your problem.

I want Renovate to create it's pull request against a custom branch other that the repository's default branch.
Is there an intended way to let renovate merge into a custom branch?

We suppose this to be the most convenient way for our use case, related to CI.
In particular, every time one of our developers pushes to the main branch, renovate starts to rebase all of the update-branches, which again starts a new cloudbuild for every PR since google cloudbuild listens on every push.

Since I haven't found a way to minimize the amount of pushes by  renovate the easiest is to create a new branch for renovate to merge into, which we can merge with master (both ways) from time to time.

**Comment by rarkins** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#basebranches

**Reply by the-voidl** [reply]
Found a typo in my base config... that's why this didn't work. Thanks anyway
